.travis.yml

```diff
@@ -7,8 +7,7 @@ services:
 python:
 - 3.8
 install:
-# Make a wheel and install it to test to catch possible
-# issues with releases
+# Make a wheel and install it
```

Comment:
Can we delete this file? I'm surprised travis still works! It looks like everything is in the GitHub workflow:

https://github.com/jupyterhub/repo2docker/blob/45229ad613a8e88c4cfb8cf60da6e9551824427d/.github/workflows/test.yml#L57-L59

https://github.com/jupyterhub/repo2docker/blob/45229ad613a8e88c4cfb8cf60da6e9551824427d/.github/workflows/test.yml#L106-L110

Reply:
Done!